changelogs/fragments/543-qos-buffer-bugfix.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic_qos_buffer - Fix QoS buffer bug (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/543).
```

A more detailed description would be more helpful to users reading the release notes.

```suggestion
  - sonic_qos_buffer - Modify buffer profile handling to match new CVL requirements (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/543).
```